src/domains/tools/email_tool.py

```
@@ -0,0 +1,32 @@
+    body: str
+    sender: str = ""
+    cc: List[str] = None
+    bcc: List[str] = None
+    attachments: List[str] = None
+    html_body: str = ""
+
+
+class SendEmailTool(BaseTool):
+    name: str = "send_email"
+    description: str = "Send an email to a recipient"
+    args_schema: Type[BaseModel] = EmailDetails
```

Ruff (0.12.2)

Undefined name `Type`

(F821)